Formula/t/terramaid.rb

```diff
@@ -0,0 +1,42 @@
+class Terramaid < Formula
+  desc "Utility for generating Mermaid diagrams from Terraform configurations"
+  homepage "https://github.com/RoseSecurity/Terramaid"
+  url "https://github.com/RoseSecurity/Terramaid/archive/refs/tags/v1.6.2.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/RoseSecurity/Terramaid/archive/refs/tags/v1.6.3.tar.gz"
```

We can bump this to latest

Reply:
Great, thanks!